src/main.rs

```diff
@@ -67,21 +69,33 @@ struct Opt {
 
     /// Download recipes and items from the GW2 API, replacing any previously cached recipes and items
     #[structopt(long)]
-    reset_cache: bool,
+    reset_data: bool,
 
     #[structopt(long, parse(from_os_str), help = &CACHE_DIR_HELP)]
     cache_dir: Option<PathBuf>,
 
+    #[structopt(long, parse(from_os_str), help = &DATA_DIR_HELP)]
+    data_dir: Option<PathBuf>,
+
     #[structopt(long, parse(from_os_str), help = &CONFIG_FILE_HELP)]
     config_file: Option<PathBuf>,
 }
 
 static CACHE_DIR_HELP: Lazy<String> = Lazy::new(|| {
+    format!(
+        r#"Save cached API calls to this directory
+
+If provided, the parent directory of the cache directory must already exist. Defaults to '{}'."#,
+        cache_dir(&None).unwrap().display()
+    )
+});
+
+static DATA_DIR_HELP: Lazy<String> = Lazy::new(|| {
     format!(
         r#"Save cached recipes and items to this directory
 
 If provided, the parent directory of the cache directory must already exist. Defaults to '{}'."#,
-        cache_dir().unwrap().display()
+        data_dir(&None).unwrap().display()
```

Comment:
I wonder if an additional directory is necessary? We could still call the reset option `reset_data` even with a single `cache_dir` option, and maybe doc it as something like `Cache API responses, including recipe and item data, in this directory `

Reply:
Yes, an additional directory _is_ necessary. For users like myself who backup files, it is very important to put the kind of data into the location it matches. Automatic backup scripts can then store important files (config), and useful files with a lower priority (such as these data files), while ignoring cache files.

The `dirs` module makes it trivial to do the right thing here, across every system. I cannot in good conscience not fix this while touching the code.

Reply:
I would also note that on Windows, these two directories are the same, if we use `data_local_dir`. Really, the question is not "how can we keep all the data in the same place?" and more "how valuable do we think this data is?" - then use the corresponding directory for the data.

Reply:
For me the files being stored in the data directory are also cache files, but it's fine